site2/docs/concepts-messaging.md

```diff
@@ -280,6 +293,82 @@ message = consumer.receive();
 consumer.acknowledge(message);
 ```
 
+### Retry letter topic
+
+Retry letter topic allows you to store the messages that failed to be consumed and retry consuming them later. With this method, you can custom the interval to retry the delivery of the messages, and subscribers automatically subscribe to the retry letter topic and re-consume it. When the specified maximum number of retries is reached, the unconsumed messages are moved to a [dead letter topic](#dead-letter-topic) for manual processing.
```

Comment:
Please review the grammatical errors and update. Thanks!

Reply:
thx @david-streamlio , fixed